src/server/PostManager.js

```diff
@@ -1,6 +1,34 @@
+import db from '../server/database';
+import {KeyUtils, ENTITY} from '../server/key-utils';
 
 export default class PostManager {
-  constructor() {
-
+  static addCreatedPost(entity, itemKey, userKey, timeHash, cb) {
```

Comment:
It means item key that is created or saved by user. It's too ambiguous?
Would you recommend one?